### ISSUE TITLE
Updater missing BitBucket tab from Settings

### ISSUE THREAD

**Issue by renaissancejames** [afragen/github-updater#580]
Hi,

I have installed this and installed a theme from a private bitbucket repo.

As soon as I activate the theme it works but it will show an Error 403.
So I go into the settings and the Bitbucket tab is missing from the settings.
If I deactivate that theme it reappears, I enter my login details and it accepts it, If there is an update I can then update, and then as soon as I reactivate it throws the error again.

I have looked at setting up the webhooks and get a response of:
```
{
  "message": "Theme not found or not updatable with GitHub Updater: perfect-cosmetics-theme",
  "error": true
}
```

I have gone throuhg all the instructions and cannot work out what I am missing to get this to work.

Cheers
James

**Comment by afragen** [comment]
Can you show the theme's headers?

**Comment by afragen** [comment]
Where do you see these errors? In the dashboard in Bitbucket's webhook area?

**Comment by jameskelly** [comment]
/*!
Theme Name: Perfect Cosmetics Theme Master
Theme URI: http://underscores.me/
Author: Renaissance Creative
Author URI: http://www.renaissance-creative.co.uk
Description: This is the master theme for the 2017 Perfect Cosmetics Sites
Version: 1.1.0
License: GNU General Public License v2 or later
License URI: LICENSE
Text Domain: perfect-cosmetics-theme
Tags:
Bitbucket Theme URI: /renaissancecreative/perfect-cosmetics-theme

*/

**Comment by jameskelly** [comment]
And see the errors in bitbuckets webhook, when I curl, or visit the url direct

**Comment by afragen** [comment]
Can you show a screenshot of the GHU Settings and the Bitbucket tab?

**Comment by afragen** [comment]
Just for fun can you remove the empty `Tags` header in your theme headers?

If your theme doesn't properly show in the dashboard settings, using the webhook will not work.

**Comment by jameskelly** [comment]
![screen shot 2017-08-02 at 20 38 49](https://user-images.githubusercontent.com/444163/28891555-d2629008-77c2-11e7-8c59-f2066356c186.png)
![screen shot 2017-08-02 at 20 39 26](https://user-images.githubusercontent.com/444163/28891553-d2616926-77c2-11e7-8304-9819a73ac22d.png)
![screen shot 2017-08-02 at 20 40 03](https://user-images.githubusercontent.com/444163/28891554-d2624238-77c2-11e7-9fe1-59f4b3f9ec49.png)

To get the bitbucket tab to show I had to deactivate the theme

**Comment by afragen** [comment]
That makes no sense. Are you sure you username/password is correct?

Can you refresh cache with the theme deactivated? Any errors?

**Comment by jameskelly** [comment]
with the theme inactive and i refresh the cache no errors

**Comment by afragen** [comment]
Whether or not the theme is active has no bearing on what GitHub Updater does. 

Has this ever worked correctly?

Is the 401 error notice because you don't have a GitHub personal access token set?

Can you give `read` access to both of the above themes to `afragen`? I'd like to see if I can replicate. 

What version of GHU are you using?

**Comment by jameskelly** [comment]
It is a parent/child package, but I have just removed the child option and its still not working.
Wierdly, when I made the parent the active theme, I could view the bitbucket tab, but it only showed the child, and when I removed the child from the website it no longer appeared.

I have also just added this to a new installation and it is showing the same response, Im trying to see if there is a code conflict in my theme compared to the plugin

I have this working on another site, using a different theme. 

Also, added debug on, and get another error of:

Notice: Undefined property: stdClass::$dot_org in /wp-content/plugins/github-updater-develop/src/GitHub_Updater/Settings.php on line 951

**Comment by afragen** [comment]
I'll need `read` access to both repos please/thanks.

**Comment by afragen** [comment]
I have been able to install and activate both themes independently, though I did get a PHP fatal for an undefined function `get_woocommerce_currency()` in 2 locations `acf_perfect.php` and `functions.php`. After commenting out those lines everything worked as expected. I was always able to see the Bitbucket tab and I have no other Bitbucket repos loaded.

Unfortunately, I am unable to replicate this. I am using the latest version of GitHub Updater.

Though it may not make a difference. Can you rename the GitHub Updater slug to `github-updater`.

**Comment by jameskelly** [comment]
Hi,
Thanks for your help on this, I have tried those steps and none of that worked.
I have then looked at disabling plugins and found that one that is on this site of:

https://wordpress.org/plugins/autoresponder/

**Comment by afragen** [comment]
Did disabling autoresponder make it work?

**Comment by jameskelly** [comment]
Yes. I disabled all the plugins and then deactivated each one individually and the only one that caused issues was this, I have looked the the code of this plugin thinking it might have a matching function or even use this for updating and had a token set but was unable to locate that

I am going to spend more time going through the plugin and comparing the functions to hopefully locate the issue

**Comment by afragen** [comment]
I'm going to close this issue. Please reopen if there's something for me to fix in GitHub Updater.

**Comment by afragen** [comment]
Like related to #586

**Comment by afragen** [comment]
@jameskelly I think this is fixed in the latest `develop` branch. Can you test please?

**Comment by afragen** [comment]
This should be fixed in latest develop branch in v7.3.0. Please reopen this issue if not.
